.vscode/settings.json

```diff
@@ -37,5 +37,7 @@
             "pattern": "**/*.spec.ts",
             "command": "yarn test:theia"
         }
-    ]
-}
\ No newline at end of file
+    ],
+    "editor.tabSize": 4,
+    "files.insertFinalNewline": true,
```

Comment:
It is not true for most of our JSON files. What is the motivation to force it now?

Reply:
It is well-known convention to have a newline at end-of-file, almost all editors add it automatically, and even git complains when you don't have one.  I think it would make sense to follow it.